packages/metro/src/DeltaBundler/Serializers/plainJSBundle.js

```diff
@@ -42,6 +43,7 @@ function plainJSBundle(
     ...getAppendScripts(entryPoint, graph, options),
   ]
     .filter(isJsModule)
+    .filter(options.processModuleFilter)
```

Comment:
This needs to be done to [each of the serializers](https://github.com/facebook/metro/tree/master/packages/metro/src/DeltaBundler/Serializers) to have consistent results no matter the type of output that the user is generating (it's specially important to update the sourcemap serializers to not end up with incorrect sourcemaps).